posthog/templates/bottom-notice.html

```diff
@@ -0,0 +1,19 @@
+{% if debug %}
+    <div id="debug-bar" class="bottom-notice">
+        <div><span>Current branch: </span><b><code>{{ git_branch|default:"unknown" }}</code></b><span>.</span></div>
+        <div><span>PostHog running in </span><b><code>DEBUG</code></b> mode!</div>
+        <div><span>Current revision: </span><b><code>{{ git_rev|default:"unknown" }}</code></b><span>.</span></div>
+        <button title="Close debug bar" onclick="document.getElementById('bottom-notice').remove()">
+            <code>×</code>
+        </button>
+    </div>
+{% else %}
+    <div id="tls-bar" class="bottom-notice" style="display: none">
+        <div>PostHog dangerously running in&nbsp;<b><code>PRODUCTION</code></b>&nbsp;mode without&nbsp;TLS! Use a&nbsp;valid TLS&nbsp;certificate and&nbsp;route&nbsp;to&nbsp;<code>https://</code>.</div>
+    </div>
+    <script>
+        const bar = document.getElementById('tls-bar')
+        if (location.protocol !== 'https:') bar.style = ''
```

Comment:
Looks cool, just this comment:

> Styles should not be set by assigning a string directly to the style property (as in elt.style = "color: blue;"), since it is considered read-only. [MDN](https://developer.mozilla.org/en-US/docs/Web/API/ElementCSSInlineStyle/style)

I think setting `bar.style.display = 'block'` is the better way to go.

Alternatively, some `if not https then document.body.appendChild(warningDiv)` style approach could work as well.

Reply:
Should be more elegant now!